core/src/trezor/ui/layouts/tt.py

```diff
@@ -50,58 +60,105 @@
     "confirm_metadata",
     "confirm_replacement",
     "confirm_modify_fee",
+    "confirm_coinjoin",
+    "not_cancelled",
 )
 
 
-def confirm_action(
+async def confirm_action(
     ctx: wire.GenericContext,
     br_type: str,
     title: str,
-    action: str,
+    action: str = None,
     description: str = None,
-    verb: Union[str, bytes] = Confirm.DEFAULT_CONFIRM,
-    icon: str = None,
+    description_param: str = None,
+    description_param_font: int = ui.BOLD,
+    verb: Union[str, bytes, None] = Confirm.DEFAULT_CONFIRM,
+    verb_cancel: Union[str, bytes, None] = Confirm.DEFAULT_CANCEL,
+    hold: bool = False,
+    icon: str = None,  # TODO cleanup @ redesign
+    icon_color: int = None,  # TODO cleanup @ redesign
+    reverse: bool = False,  # TODO cleanup @ redesign
+    larger_vspace: bool = False,  # TODO cleanup @ redesign
     br_code: EnumTypeButtonRequestType = ButtonRequestType.Other,
     **kwargs: Any,
```

Comment:
this is now unused

Reply:
Fixed in 28f676ca68135ca4a6b46cdf7b18815a361e4b36.